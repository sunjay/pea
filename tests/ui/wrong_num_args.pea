fn foo(a, b) {
    let q = a * 3 - b;
    println!(q);
    println!(b);
}

fn main() {
    let x = foo;

    // Wrong number of arguments!!!
    x(3);
}
