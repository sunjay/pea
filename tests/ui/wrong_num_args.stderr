stack backtrace:
   0: main
error: wrong number of arguments for function `foo`: expected 2, got 1
